### PR TITLE
Add trailing '.' to PATCH query generation

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Fixes issue https://github.com/solid/solid-client/issues/149

### CODE REVIEW

File: test/unit/web-util-test.js [hunk 1]
```diff
@@ -6,8 +6,19 @@ var webUtil = require('../../src/util/web-util')
...
+    webUtil.composePatchQuery(toDelLinks, toAddLinks),
+    expectedQuery
+  )
+  t.end()
+})
+
+test('web-util.composePatchQuery() - single statement', function (t) {
+  let toDelLinks = []
+  let toAddLinks = ['triple1']
+  let expectedQuery = 'INSERT DATA { triple1 . };\n'
```
Comment (deiu) [inline test/unit/web-util-test.js]: I'm pretty sure that `triple1` is not valid Turtle either. :)

Comment (dmitrizagidulin) [reply test/unit/web-util-test.js]: @deiu ha! probably. But would you really expect the compose query function to run it through some sort of Turtle validator?

File: src/util/web-util.js [hunk 2]
```diff
@@ -52,14 +52,14 @@ function composePatchQuery (toDel, toIns) {
     toDel = toDel.map(function (st) {
       return statementToNT(st, excludeDot)
```
Comment (deiu) [inline src/util/web-util.js]: What's the point of calling `excludeDot` every time? All NT statements end with a dot.

File: src/util/web-util.js [hunk 3]
```diff
@@ -52,14 +52,14 @@ function composePatchQuery (toDel, toIns) {
...
     })
-    query += 'DELETE DATA { ' + toDel.join(' . ') + ' };\n'
+    query += 'DELETE DATA { ' + toDel.join(' . ') + ' . };\n'
```
Comment (deiu) [inline src/util/web-util.js]: So you're calling `excludeDot` just so you can append a dot later? What's the point of this code?

**Review by deiu — CHANGES_REQUESTED** [review]
Bad.

File: src/util/web-util.js (hunk 3 above)
Comment (dmitrizagidulin) [reply src/util/web-util.js]: (will remove it)

**Comment by dmitrizagidulin** [comment]
@deiu Removed.

**Review by deiu — APPROVED** [review]

**Comment by deiu** [comment]
The code looks good. FYI, the tests miss prefix definition for `ex:`. Alternatively, you can just use absolute URIs.

Anyways, 👍 from me.

**Pull request merged**